README.md

```diff
@@ -13,6 +13,7 @@ Options
 ----
 * **hash** - [Optional] A unicode string that will be used to generate the image. Defaults to a random hash based on the current time.
 * **options** - [Optional] An options object used to customize the generated image.
+    * **foreground** - The foreground color expressed as an rgba value array to use for the image foreground. For example, use [255,0,0,255] for red. Defaults to a color calculated from hash.
```

Comment:
Can we reword this to make it more clear that the foreground is normally derived from the hash? Something like:
The foreground color is automatically derived from the hash value. Use this option to override that behavior and provide a rgba value array instead (e.g. [255,0,0,255] for red).